Add wss4j dependency and support info to related document (#340)

EmreKp opened the pull request:
Hello,

I updated documentation and submitted PR related with #340 . I also pasted the note for support of which interceptors, may be helpful for other developers.

Thanks..

shumonsharif commented on docs/modules/ROOT/pages/ws-security.adoc:
```diff
@@ -3,6 +3,19 @@
 
 The CXF framework's https://cxf.apache.org/docs/ws-security.html[WS-Security] implementation is based on https://ws.apache.org/wss4j/user_guide.html[WSS4J]. The example below shows how to integrate with WSS4J using interceptors.
 
+To use WSS4J interceptors in your code, firstly you should add CXF's WSS4J dependency (Replace `version` with CXF version used by Quarkus).
```
- You are adding CXF's **WS-Security** dependency _(not WSS4J)_
- I don't believe we need to say _"(Replace `version` with CXF version used by Quarkus)"_ .... for example, if someone is using the CXF BOM, then they don't need a version specified here at all.  Would suggest we remove this statement, along with the version tag in the XML block.  A basic understanding of Maven can be expected of the extension users, so I think we can omit this level of detail.

EmreKp replied:
Thanks for suggestions, I changed version statement like "_you should add latest version_"  to satisfy any curiosities,  if it needs better statement, please let me know.

shumonsharif reviewed (changes requested):
Thanks @EmreKp for this PR.  Would you kindly make the suggested updates?

EmreKp commented:
Thanks for suggestions, I also changed version statement like "_you should add latest version_" to answer any questions, if it needs better statement, please let me know.

shumonsharif commented:
> Thanks for suggestions, I also changed version statement like "_you should add latest version_" to answer any questions, if it needs better statement, please let me know.

Thanks @EmreKp Yes, I would suggest we remove that statement all together; adding the latest version is not accurate either.

EmreKp commented:
> > Thanks for suggestions, I also changed version statement like "_you should add latest version_" to answer any questions, if it needs better statement, please let me know.
> 
> Thanks @EmreKp Yes, I would suggest we remove that statement all together; adding the latest version is not accurate either.

I removed it for now, maybe it can be added later if some users would have problem concerned with that.
